packages/generators/init-template/default/package.json.js

```diff
@@ -1,6 +1,9 @@
 module.exports = (isUsingDevServer) => {
     const scripts = {
         build: 'webpack --mode=production',
+        'build:dev': 'webpack',
+        'build:prod': 'webpack --mode=production',
+        watch: 'webpack --watch',
```

Comment:
@webpack/cli-team What do you think better to use `webpack --watch` or `webpack watch` (i.e. flag vs command)

Reply:
I'd prefer it as a flag since it doesn't have any options AFAIK

Reply:
I'm ok with `--watch`